.cirrus.yml

```
@@ -307,7 +307,16 @@ in_podman_task:
         # This is key, cause the scripts to re-execute themselves inside a container.
         IN_PODMAN: 'true'
         BUILDAH_ISOLATION: 'chroot'
-        STORAGE_DRIVER: 'vfs'
+    matrix:
+        - env:
+              STORAGE_DRIVER: 'vfs'
+        - env:
+              STORAGE_DRIVER: 'overlay'
+    matrix:
+        - env:
+              NESTED_STORAGE_DRIVER: 'vfs'
+        - env:
+              NESTED_STORAGE_DRIVER: 'overlay'
```

I believe cirrus will happily accept this invalid YAML, but I believe `hack/get_ci_vm.sh` will puke on it because it uses python-yaml to parse the file.  If this functionality is important, I believe something like the following would work:

```suggestion
    matrix:
        - env:
              STORAGE_DRIVER: 'vfs'
              NESTED_STORAGE_DRIVER: 'vfs'
        - env:
              STORAGE_DRIVER: 'overlay'
              NESTED_STORAGE_DRIVER: 'vfs'
        - env:
              STORAGE_DRIVER: 'vfs'
              NESTED_STORAGE_DRIVER: 'overlay'
        - env:
              STORAGE_DRIVER: 'overlay'
              NESTED_STORAGE_DRIVER: 'overlay'
```

---

Done.